packages/central-server/src/apiV2/import/importEntities/updateEntities.js

```diff
@@ -130,12 +131,27 @@ export async function updateCountryEntities(
         `Entity code '${code}' is not unique`,
         excelRowNumber,
         'code',
-        countryName,
+        country?.name || NO_COUNTRY,
       );
     }
     codes.push(code);
-    const { parentGeographicalArea, parentEntity } =
-      (await getOrCreateParentEntity(transactingModels, entityObject, country, pushToDhis)) || {};
+    let parentGeographicalArea;
+    let parentEntity;
+
+    // Entities without a country (ie. projects) do not have parent entities
+    if (country) {
+      const parentData = await getOrCreateParentEntity(
+        transactingModels,
+        entityObject,
+        country,
+        pushToDhis,
+      );
+      if (parentData) {
+        parentGeographicalArea = parentData.parentGeographicalArea;
+        parentEntity = parentData.parentEntity;
+      }
```

Comment:
`getOrCreateParentEntity` should always return value, do you think it is better to throw error rather than return null in `getOrCreateParentEntity`?